haskell/src/Monocle/Prelude.hs

```diff
@@ -426,8 +422,13 @@ inText sub txt = case indices sub txt of
   [] -> False
   _ -> True
 
+monocleLogIO :: Text -> IO ()
+monocleLogIO log = liftIO $ withLogger callLogger
+  where
+    callLogger logger = doLog logger $ encodeUtf8 @Text @ByteString . from $ log
+
```

Comment:
We shouldn't need the unlifted version, and we might want to keep the bytestring version. How about:

```suggestion
-- | Print a message to the stderr with a timestamp
logMessage :: MonadIO m => ByteString -> m ()
logMessage msg = liftIO $ withLogger (flip doLog msg)

-- | Print a text message to the stderr
logText :: MonadIO m => Text -> m ()
logText = logMessage . from
```

Reply:
Thanks you ! done